src/v3/test/integration/unlock-account-success.test.tsx

```diff
@@ -35,4 +40,35 @@ describe('unlock-account-success', () => {
       ...createAuthJsPayloadArgs('POST', 'idp/idx/cancel'),
     );
   });
+
+  describe('useInteractionCodeFlow', () => {
```

Comment:
todo: add tests for useInteractionCodeFlow for other affected tests with back to sign in link